src/components/slider/context/SliderGroup.tsx

```diff
@@ -11,40 +12,24 @@ interface SliderGroupProps {
   children?: React.ReactNode;
 }
 
-interface SliderGroupState {
-  value: tinycolor.ColorFormats.HSLA;
-}
-
-export default class SliderGroup extends Component<SliderGroupProps, SliderGroupState> {
-  static displayName = 'IGNORE';
+const SliderGroup = (props: SliderGroupProps) => {
+  const {color, onValueChange, children} = props;
+  const [value, setValue] = useState(Colors.getHSL(color));
 
-  constructor(props: SliderGroupProps) {
-    super(props);
+  const valueSetter = useCallback((value: tinycolor.ColorFormats.HSLA) => {
```

Comment:
'valueSetter' is not a proper name for a callback. We usually use the name of the prop it is passed to or as an action describing what this method does

Reply:
Is _setValue better?

Reply:
Yes